cadc-access-control-admin/config/ac-admin-email.properties

```diff
@@ -27,7 +34,11 @@
 ###
 
 smtp.host=example.host
-smtp.port=587
+smtp.port=25
+smtp.auth.host=example-auth.host
+smtp.auth.port=587
+smtp.account=account@example.host
+smtp.password=changeme
```

Comment:
maybe make these `smtp.auth.account` and `smtp.auth.password` so it's clear the 4 properties are grouped together?